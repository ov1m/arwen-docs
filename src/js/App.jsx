import React, { Component } from 'react';
import Header from './components/base/header';

class App extends Component {
	constructor() {
		super();

		this.state = {
			page: 'home'
		}
	}

	render() {
		return (
			<div>
				<Header />
				12xzcasdas
			</div>
		);
	}
}

export default App;
