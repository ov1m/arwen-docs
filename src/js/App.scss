@import '../style/variables.scss';
@import '../style/reset.scss';
@import '../style/base.scss';

#app {
  display: flex;
  min-height: 100%;
}

.wrapper {
  flex: 1;
  background: rgba(0, 0, 0, 0.075);
}

.content {
  padding: 0 4rem;
}