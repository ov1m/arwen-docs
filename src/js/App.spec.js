/*
 */

import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import App from './app';

it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
