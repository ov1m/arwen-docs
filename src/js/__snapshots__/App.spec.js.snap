// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <nav
    className=""
  >
    <li>
      <a
        href="#asdas"
      >
        Getting Started
      </a>
    </li>
  </nav>
  12xzcasdas
</div>
`;
